bigquery_etl/glam/templates/macros.sql

```diff
@@ -31,6 +31,9 @@ static_combos AS (
         table.* EXCEPT(
             {{ cubed_attributes | join(",") }}
         ),
+        {% if sampled %}
+            {{ sampled_exp }},
+        {% endif %}
```

Comment:
- It seems like you could just use `sampled_exp`.
- Having the calling code set the column alias seems a little sketchy.
```suggestion
        {% if sampled_exp %}
            {{ sampled_exp }} AS sampled,
        {% endif %}
```

Reply:
Yes to all!